BraveRewardsUI/Settings/Tips/Details/TipsDetailViewController.swift

```diff
@@ -94,15 +123,18 @@ extension TipsDetailViewController: UITableViewDataSource, UITableViewDelegate {
     guard let typedSection = Section(rawValue: section) else { return 0 }
     switch typedSection {
     case .summary:
-      return 1
+      return 1 + (tipsList.filter({$0.category == .recurringTip}).isEmpty ? 0 : 1 )
```

Comment:
lint/format: un-needed enclosing parenthesis

Reply:
<img width="1088" alt="Screenshot 2019-06-20 at 10 03 07 PM" src="https://user-images.githubusercontent.com/5398557/59865732-6da71a00-93a7-11e9-853f-6606f3acfefb.png">
Removing them results in this error